documentation/tutorials/google.md

```diff
@@ -65,12 +65,6 @@ defmodule MyApp.Accounts.User do
       # Required if you're using the password & confirmation strategies
       upsert_fields []
       change set_attribute(:confirmed_at, &DateTime.utc_now/0)
-      change after_action(fn _changeset, user, _context ->
```

Comment:
This isn't quite complete. They should put the action into `auto_confirm_actions` which allows them to remove the comment and anything underneath it.

Reply:
You're right. I'm working on a doc overhaul for AA so this will definitely be addressed.